velox/functions/lib/Re2Functions.cpp

```diff
@@ -30,6 +33,8 @@ using ::facebook::velox::exec::VectorFunction;
 using ::facebook::velox::exec::VectorFunctionArg;
 using ::re2::RE2;
 
+static const int kMaxCompiledRegexes = 20;
```

Comment:
Can we make this configurable? Does the existing workload follow this limit?

Reply:
I dont have a storng opinion but i think @mbasmanova  does, as she answered bellow so lets keep it as to move fast and re iterate if needed

Reply:
It makes sense to cap this to a constant as RE2 compilation times can be large. But we should make sure this is enough to help some of the queries which currently fail with this error. Have we tested this change with those queries to see if its enough?